builder_data.yaml

```diff
@@ -45,7 +45,7 @@ yum_staging_server: weth.delivery.puppetlabs.net
 yum_repo_path: "/opt/repository/yum"
 nonfinal_yum_repo_path: "/opt/repository-nightlies/yum"
 apt_archive_path: "/opt/release-archives-staging/apt"
-freight_archive_path: "/opt/freight-archives-staging"
+freight_archive_path: "/opt/freight-archives"
```

Comment:
the path I'm using in puppetlabs-modules is `/opt/tools/freight-archives`.

Reply:
whoops, updated